packages/perseus/src/widgets/number-line/number-line.tsx

```diff
@@ -160,34 +160,18 @@ const TickMarks: any = Graphie.createSimpleClass((graphie, props) => {
 
         const labelTicks = props.labelTicks;
         if (labelTicks || props.labelStyle === "decimal ticks") {
-            results.push(_label(graphie, props.labelStyle, x, x, base));
+            if (x === leftLabel || x === rightLabel) {
+                results.push(
+                    graphie.style({color: KhanColors.BLUE}, () =>
```

Comment:
Just confirming, this color has the appropriate color contrast right? The blue color looks a little light to me.

Reply:
Okay I checked the color out and it does pass AA so we should be good, I would love for it to be darker for the type of text it it. But non-blocking 😉
![image](https://github.com/user-attachments/assets/0d7bddf2-1d80-4eae-b4cc-aa66ade43558)

Reply:
Agreed! That color was the first thing I checked, too.